Testing/load/timeline_load_k6.js

```diff
@@ -42,7 +42,7 @@ let users = [
   { username: "loadtest_10", password: passwd, hdid: null, token: null, refresh: null, expires: null },
   { username: "loadtest_11", password: passwd, hdid: null, token: null, refresh: null, expires: null },
   { username: "loadtest_12", password: passwd, hdid: null, token: null, refresh: null, expires: null },
-//  { username: "loadtest_13", password: passwd, hdid: null, token: null, refresh: null, expires: null },
```

Comment:
NIT:  Adding additional code or other changes should be documented in the PR.

Reply:
I am not sure how this go updated. it wasn't in my change log

Reply:
not sure how this change floated in.. it was the same before.